SimPEG/electromagnetics/base_1d.py

```diff
@@ -0,0 +1,947 @@
+import properties
+from scipy.constants import mu_0
+import numpy as np
+from scipy import sparse as sp
+from empymod.transform import get_dlf_points
+
+from ..data import Data
+from ..simulation import BaseSimulation
+
+# from .time_domain.sources import MagDipole as t_MagDipole, CircularLoop as t_CircularLoop
+# from .frequency_domain.sources import MagDipole as f_MagDipole, CircularLoop as f_CircularLoop
+
+from .. import utils
+from .. import props
+from empymod.utils import check_hankel
+
+try:
```

Comment:
Isn't multiprocessing an included fundamental python package? What environments have you seen that hasn't had multiprocessing? It's likely safe to just use `from multiprocessing import Pool` without any special guards.

Reply:
sounds good. I'll consider `multiprocessing` as a base python package.